### PR TITLE
PR review changes

### PR DESCRIPTION
**Pull request by cdwilson**
These are some additional changes to https://github.com/golioth/thingy91-golioth/pull/5

### CODE REVIEW

File: src/app_work.c [hunk 1]
```diff
@@ -234,7 +233,8 @@ void user_led_set(uint8_t state)
 }
 
 /* Formatting string for sending sensor JSON to Golioth */
-#define JSON_FMT "{\
+#define JSON_FMT                                                                                   \
+	"{\
 \"weather\":{\"tem\":%d.%d,\"pre\":%d.%d,\"hum\":%d.%d,\"gas\":%d.%d},\
 \"light\":{\"red\":%d,\"blue\":%d,\"green\":%d,\"ir\":%d},\
 \"accel\":{\"x\":%d.%d,\"y\":%d.%d,\"z\":%d.%d}\
```
Comment (szczys) [inline src/app_work.c]: Should lines 238-240 also be indented?

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 * Add a `CHANGELOG.md` to track changes moving forward.
 * Merge changes from [`golioth/reference-design-template@template_v1.2.0`](https://github.com/golioth/reference-design-template/tree/template_v1.2.0)
+* Added debug logging for counters
```
Comment (szczys) [inline CHANGELOG.md]: I think a best practice would be to update the Changelog in the same commit as the change. Do you think we can squash this commit?

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,22 +12,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 ### Added
 
 * Add a `CHANGELOG.md` to track changes moving forward.
-* Merge changes from [`golioth/reference-design-template@template_v1.2.0`](https://github.com/golioth/reference-design-template/tree/template_v1.2.0)
-* Added debug logging for counters
+* Merge changes from [`golioth/reference-design-template@template_v1.2.0`](https://github.com/golioth/reference-design-template/tree/template_v1.2.0).
```
Comment (szczys) [inline CHANGELOG.md:15]: Can changes in this commit be rebased onto where the lines were originally added?

**Comment by cdwilson** [comment]
@szczys good feedback, sorry for the mess with the CHANGELOG being all over the place.

I've cleaned up all the commits so that they should be easier to read one-by-one, and the CHANGELOG entries are also associated with the correct commit now.

**Review by szczys — APPROVED** [review]
LTGM, thanks for all the clean up!

**Pull request merged**